src/Protocol/Protocol_1_9.cpp

```diff
@@ -2524,7 +2524,15 @@ void cProtocol_1_9_0::HandlePacketBoatSteer(cByteBuffer & a_ByteBuffer)
 void cProtocol_1_9_0::HandlePacketChatMessage(cByteBuffer & a_ByteBuffer)
 {
 	HANDLE_READ(a_ByteBuffer, ReadVarUTF8String, AString, Message);
-	m_Client->HandleChat(Message);
+	if ((Message.size()) > MAX_CHAT_MSG_LENGTH)
+	{
+		m_Client->Kick(std::string("Please don't exceed the maximum message length of ")
```

Comment:
Shouldn't this be in `cClient::HandleChat` so that is protocol version agnostic?

Reply:
yes, it should. changing this now